docs/partners/rest-api/README.md

```diff
@@ -0,0 +1,17 @@
+# Partners REST API Endpoint Documentation
+
+An alternative to manually using the [Jetpack Partner Portal](#) is to query the REST API endpoints that it uses.
+
+## Authentication
```

Comment:
Since the partner portal is still internal only, should we provide or link to documentation on how to generate an oauth token outside of the partner portal?

Reply:
That's probably a good idea. 👍 

Although I have no idea how to do that. :)